res/controllers/engine-api.d.ts

```diff
@@ -31,10 +34,50 @@ declare interface ScriptConnection {
     readonly isConnected: boolean;
 }
 
+/** JavascriptPlayerProxy */
+
+declare interface Player {
+    readonly artist: string
+    readonly title: string
+    readonly album: string
+    readonly albumArtist: string
+    readonly genre: string
+    readonly composer: string
+    readonly grouping: string
+    readonly year: string
+    readonly trackNumber: string
+    readonly trackTotal: string
+
+    /** Emitted when the track is unloaded from the player. */
+    trackUnloaded: QtSlot<() => void>
+
+    /**
+     * The following are emitted when a nex track is loaded to the player
+     * or when the current track's metadata change.
+     */
+    artistChanged: QtSlot<(newArtist: string) => void>
+    titleChanged: QtSlot<(newTitle: string) => void>
+    albumChanged: QtSlot<(newAlbum: string) => void>
+    albumArtistChanged: QtSlot<(newAlbumArtist: string) => void>
+    genreChanged: QtSlot<(newGenre: string) => void>
+    composerChanged: QtSlot<(newComposer: string) => void>
+    groupingChanged: QtSlot<(newGrouping: string) => void>
+    yearChanged: QtSlot<(newYear: string) => void>
+    trackNumberChanged: QtSlot<(newTrackNumber: string) => void>
+    trackTotalChanged: QtSlot<(newTrackTotal: string) => void>
```

Comment:
I wrote a global documentation for those I don't think it's necessary to document them individually?

Reply:
It only appear in the editor/IDE if you document it per item. This file is not intended for human readers that read it from top to down.
This way the user sees it as 
![grafik](https://github.com/user-attachments/assets/118d6225-9df4-4d3e-819d-f2d020994971)
but only for artistChanged.

Please add documentation for each item therefore, as in the rest of the typescript declaration files.